### DISCUSSION TITLE
Looks really cool

### DISCUSSION THREAD

**Discussion by ZoCanada** [DaXcess/LCVR#26] [General]
Been watching this mod closely, my friends and I are really excited to see what comes of the initial release!

To help temper our expectations, I was wondering what kind of multiplayer functionality we can expect. Less interested in whether we'll be able to see each other's ik data and more interested in whether we'll be able to connect to each other at all.

**Comment by AbandonedPlayer** [comment]
There is ongoing discussions about multiplayer and progress on this build on discord. 
https://flat2vr.com/
https://discord.com/channels/747967102895390741/1173840916788809779/1189878630512332811
